app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java

```diff
@@ -184,4 +192,38 @@ public void run() {
             }
         });
     }
+
+    @SuppressLint("ResourceType")
+    public boolean howToConnectDialog(String title, String message, int iconID) {
+        if (!tvData.contains(title)) {
+            try {
+                final AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
+                LayoutInflater inflater = getLayoutInflater();
+                View dialogView = inflater.inflate(R.layout.sensor_custom_dialog, null);
+                builder.setView(dialogView);
+                builder.setTitle(title);
+                final TextView tv = (TextView) dialogView.findViewById(R.id.custom_dialog_text);
+                final ImageView iv = (ImageView) dialogView.findViewById(R.id.sensor_schematic);
+                tv.setText(String.format("Connect the %s sensor pins to the PSLab device as shown below", message));
+                iv.setImageResource(iconID);
+
+
```

Comment:
@Vikum94  Please remove this extra spaces

Reply:
Formated code :+1: